crawl-ref/source/potion.cc

```diff
@@ -73,7 +69,11 @@ class PotionCuring : public PotionEffect
             return true;
         }
 
-        return false;
```

Comment:
I think this makes it impossible for players to quaff a heal_wounds manually when they're at full HP and not rotted. Maybe that's a good change, but I'm not sure it's a good idea to sneak that in with a gozag change.